packages/api-v2/src/plan/plan.controller.ts

```diff
@@ -21,13 +21,14 @@ import {
   GetPlanResponse,
   UpdatePlanResponse,
 } from "@graduate/common";
+import { EmailConfirmationGuard } from "src/guards/emailConfirmation.guard";
 
 @Controller("plans")
 export class PlanController {
   constructor(private readonly planService: PlanService) {}
 
   /** Creates a plan for the signed in user. */
-  @UseGuards(JwtAuthGuard)
+  @UseGuards(JwtAuthGuard, EmailConfirmationGuard)
```

Comment:
move the @UseGuard to above the controller, that way all routes are protected by EmailConfirmationGuard. Makes sense, cause no plan CRUD should be possible without it.